### PR TITLE
sci-electronics/kicad: Bump to version 4.0.2

### PR DESCRIPTION
**Pull request by zpuskas**
Gentoo-Bug: #574692

Package-Manager: portage-2.2.27

### CODE REVIEW

**Comment by zpuskas** [comment]
@idella 
@jlec 
@proxy-maint

**Comment by idella** [comment]
zpuskas should really be looking at selecting old versions for removal and selecting a candidate for putting up for stable.

File: sci-electronics/kicad/kicad-4.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,185 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
```
Comment (jlec) [inline sci-electronics/kicad/kicad-4.0.2.ebuild]: Why not `EAPI=6`?

File: sci-electronics/kicad/kicad-4.0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,185 @@
+# Copyright 1999-2015 Gentoo Foundation
```
Comment (jlec) [inline sci-electronics/kicad/kicad-4.0.2.ebuild]: Wrong copyright year.

File: sci-electronics/kicad/kicad-4.0.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc examples github i18n minimal python webkit"
+LANGS="bg ca cs de el es fi fr hu it ja ko nl pl pt ru sk sl sv zh_CN"
+for lang in ${LANGS} ; do
+	IUSE="${IUSE} linguas_${lang}"
+done
+unset lang
```
Comment (jlec) [inline sci-electronics/kicad/kicad-4.0.2.ebuild]: Also unset LANGS?

File: sci-electronics/kicad/kicad-4.0.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+		# remove additional demos/examples as its not strictly required to run the binaries 
+		sed -e '/add_subdirectory( demos )/d' -i CMakeLists.txt || die "sed failed"
+	fi
+
+	# Add important missing doc files
+	sed -e 's/INSTALL.txt/AUTHORS.txt CHANGELOG.txt README.txt TODO.txt/' -i CMakeLists.txt || die "sed failed"
+}
+
+src_configure() {
+	need-wxwidgets unicode
```
Comment (jlec) [inline sci-electronics/kicad/kicad-4.0.2.ebuild]: Unnecessary as all versions are unicode always.

File: sci-electronics/kicad/kicad-4.0.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,185 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit cmake-utils fdo-mime flag-o-matic gnome2-utils python-single-r1 vcs-snapshot wxwidgets versionator
```
Comment (jlec) [inline sci-electronics/kicad/kicad-4.0.2.ebuild]: `fdo-mime.eclass`is deprecated. Please use `xdg.eclass`.

**Pull request merged**